physionet-django/export/views.py

```diff
@@ -61,3 +64,49 @@ def get(self, request, project_slug, version, *args, **kwargs):
         project = get_object_or_404(PublishedProject, slug=project_slug, version=version)
         serializer = PublishedProjectDetailSerializer(project)
         return Response(serializer.data)
+
+
+class PublishedProjectSearch(mixins.ListModelMixin, generics.GenericAPIView):
+    """
+    Search for a Published Project using the get_content function inside Search Module's views.py
+    """
+
+    serializer_class = PublishedProjectSerializer
+
+    def get_queryset(self):
+        resource_type = self.request.GET.getlist('resource_type', 'all')
+        search_term = self.request.GET.get('search_term', ' ')
+
+        print(search_term, resource_type)
```

Comment:
Could you briefly explain the purpose of the print statements?

Reply:
The print statements were used while I was debugging - no need for them in production.
Removed them.